pkg/controllers/uiplugin/controller.go

```diff
@@ -251,7 +251,8 @@ func (rm resourceManager) registerPluginWithConsole(ctx context.Context, pluginI
 			OperatorSpec: operatorv1.OperatorSpec{
 				ManagementState: operatorv1.Managed,
 			},
-			Plugins: clusterPlugins,
+			Plugins:       clusterPlugins,
+			Customization: *cluster.Spec.Customization.DeepCopy(),
```

Comment:
I wonder if this is really needed, this is a last resource path we are using and we should not change other things unrelated to plugins

Reply:
I had found that the `spec.customization.customLogoFile.key` was being unset when the Patch method was called in [reconciler.go](https://github.com/rhobs/observability-operator/blob/main/pkg/reconciler/reconciler.go) for the Merger.Reconcile. This leads the Console Operator to get stuck in a reconciliation loop. This issue will pop up on all consoles using a custom image like ROSA. All this line does is ensure that the customization is available on both sides of the patch.

However, although this only happened with the customization and I checked that there wasn't some other item that was getting removed in cluster yaml, it's possible it could happen with other items. I'm not exactly certain how to go about debugging this, but I'd be happy to if someone could point me in the right direction. Otherwise I can remove it, but the UIPlugins won't deploy on ROSA and will cause the Console Operator to get stuck in a reconciliation loop.